test/logic/player/favorites_test.dart

```diff
@@ -10,46 +10,46 @@ void main() {
   final favoriteSong2 = songWith(id: 4, title: 'Song 4', isFavoriteInMediaStore: true);
   final favoriteSong3 = songWith(id: 5, title: 'Song 5', isFavoriteInMediaStore: true);
 
-  setUp(() async {
-    await setUpAppTest(() {
```

Comment:
Are we sure this is a good API change that we not longer can use setUp between tests?

Reply:
I'd argue it's not - it reduces the reusability of setup logic

In some cases it might also make that tests utilities are not composable with each other - this is a real case in tests that I had recently, when I made a not very composable API desicion, which put constraints on how tests could be written, so I had to rewrite it

Reply:
From the description of the PR I see that you want them to live inside the same zone as other test logic

It this possible to achieve in some other way?